versioned/persist/adapter/src/main/java/org/projectnessie/versioned/persist/adapter/DatabaseAdapter.java

```diff
@@ -163,6 +168,7 @@ Hash transplant(
    * @param from commit-hash to start reading commits from.
    * @param toBranch target branch to commit to
    * @param expectedHead if present, {@code toBranch}'s current HEAD must be equal to this value
+   * @param updateCommitMetadata Function to allow resetting properties application due to merge.
```

Comment:
Please reword. "resetting properties application" is confusing.

Reply:
ok